prisma/schema.prisma

```diff
@@ -424,7 +425,7 @@ model screening {
 }
 
 model student {
-  id                                        Int                                    @id @default(autoincrement())
+  id                                        Int                                    @id @default(autoincrement())wix_creation_date
```

Comment:
I'm not sure how this change got here, however it introduces a syntax error and Prisma fails to load:
```
Error: Get config: Schema Parsing P1012

error: Error validating: This line is not a valid field or attribute definition.
  -->  schema.prisma:428
   | 
427 | model student {
428 |   id                                        Int                                    @id @default(autoincrement())wix_creation_date
429 |   createdAt                                 DateTime                               @default(now()) @db.Timestamp(6)
   | 

Validation Error Count: 1
```

Removing this again and running `npx prisma format` and `npx prisma generate` fixes this and should make the build work again